js/app.js

```diff
@@ -0,0 +1,173 @@
+// Array to store currency rates
+let currencyRates = [];
+
+// function  to show message
+function displayMessage(message, isSuccess) {
+  // if no parameter given then it would be true by default
```

Comment:
You can define a default value for `isSuccess` using a [default parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)

Reply:
Thank you very much for review and suggestions!